tests/test.py

```diff
@@ -12,6 +12,9 @@
 
 def dumpFile(fname, language):
     f = path.join(BASE_DIR, "tests/", fname)
+    global TEST_DIR
```

Comment:
Maybe make TEST_DIR absolute or perform this check when TEST_DIR is initialized on the begining of file https://github.com/Nic30/hdlConvertor/blob/master/tests/test.py#L6